substrate/client/network/Cargo.toml

```diff
@@ -42,17 +46,23 @@ smallvec = "1.11.0"
 thiserror = { workspace = true }
 tokio = { version = "1.22.0", features = ["macros", "sync"] }
 tokio-stream = "0.1.7"
-unsigned-varint = { version = "0.7.1", features = ["asynchronous_codec", "futures"] }
+unsigned-varint = { version = "0.7.2", features = ["asynchronous_codec", "futures"] }
 zeroize = "1.4.3"
 prometheus-endpoint = { package = "substrate-prometheus-endpoint", path = "../../utils/prometheus" }
+prost = "0.11"
 sc-client-api = { path = "../api" }
 sc-network-common = { path = "common" }
+sc-network-types = { path = "types" }
 sc-utils = { path = "../utils" }
 sp-arithmetic = { path = "../../primitives/arithmetic" }
 sp-blockchain = { path = "../../primitives/blockchain" }
 sp-core = { path = "../../primitives/core" }
 sp-runtime = { path = "../../primitives/runtime" }
 wasm-timer = "0.2"
+litep2p = { git = "https://github.com/paritytech/litep2p", branch = "master" }
```

Comment:
I think we'll need to release litep2p on crates io, before merging this PR

Reply:
It's already released by Aaro: https://crates.io/crates/litep2p.

We could probably ask @altonen to transfer it to Parity, or re-release it under Parity umbrella.

Reply:
I can add paritytech as the owner and remove myself, that should work?

Reply:
May be keep you as the owner alongside paritytech for a while in case something goes wrong?